app/src/main/java/pl/ahyoshin/demo/AdminEndPoint.java

```diff
@@ -0,0 +1,20 @@
+package pl.ahyoshin.demo;
+
+import org.springframework.context.annotation.Scope;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@Scope(proxyMode = ScopedProxyMode.TARGET_CLASS)
+public class AdminEndPoint {
+    public AdminEndPoint() {
+    }
+
+    @PreAuthorize("hasAnyAuthority('Admin')")
+    @GetMapping({"/Admin"})
+    public void test() {
+    }
+}
+
```

Comment:
Imo testowe endpointy nie powinny trafić na develop

Reply:
mozna usunac to tylko bylo dla mnie ( :